index.js

```diff
@@ -104,6 +104,14 @@ module.exports = fp(async function (app, opts) {
 
   app.decorate('graphql', fastifyGraphQl)
 
+  fastifyGraphQl.replaceSchema = function (s) {
+    if (!s || typeof s !== 'object') {
+      throw new Error('Must provide valid Document AST')
+    }
+
+    schema = s
```

Comment:
You need to flush the AST and jit cache.

Reply:
I've updated PR and added

```js
lru.clear()
lruErrors.clear()
```

I hope, it should be correct.